app/models/status.rb

```diff
@@ -141,6 +143,11 @@ def non_sensitive_with_media?
     !sensitive? && media_attachments.any?
   end
 
+  before_validation :prepare_contents
+  before_create     :set_reblog
+  before_create     :set_visibility
+  before_create     :set_conversation
```

Comment:
The change in the logic is that there's a new set_conversation, that's what I'm saying should be `if: :local?`